client/flags.go

```diff
@@ -58,6 +59,7 @@ func PostCommands(cmds ...*cobra.Command) []*cobra.Command {
 		c.Flags().Bool(FlagAsync, false, "broadcast transactions asynchronously")
 		c.Flags().Bool(FlagJson, false, "return output in json format")
 		c.Flags().Bool(FlagPrintResponse, true, "return tx response (only works with async = false)")
+		c.Flags().Bool(FlagDryRun, false, "no action; ignore the --gas flag and perform a simulation of a transaction, but don't broadcast it")
```

Comment:
I think we can remove `no action; ` as the description gives a good enough...description 👍

Reply:
I usually aim to make command line flags docs stupid-proof :)
Removing "no action" is not a big deal though :+1:

Reply:
Oh and btw, I took inspiration from `man apt-get`:

>        -s, --simulate, --just-print, --dry-run, --recon, --no-act
>             No action; perform a simulation of events that would occur based on the current system state but do not actually change the system. [snip]

Reply:
#defeated

Reply:
LOL sorry, amended now